chat-plugins/info.js

```diff
@@ -2196,6 +2196,13 @@ var commands = exports.commands = {
 			"- !showimage <em>[url], [width], [height]</em>: shows an image to the room<br />" +
 			"<br />" +
 			"More detailed help can be found in the <a href=\"https://www.smogon.com/sim/roomauth_guide\">roomauth guide</a><br />" +
+			"<br />" +
+			"Tournament Help:<br />" +
+			"- /tour create format, type: Creates a new tournament in the current room.<br />" +
```

Comment:
Follow the convention for the rest of this. `<em>[format], [elimination|roundrobin]</em>`